src/App.vue

```diff
@@ -611,11 +596,11 @@ export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
 
   /**
    * Gets current account from object in session storage.
-   * Waits up to 10 sec for current account to be synced (typically by SbcHeader).
+   * Waits up to 5 sec for current account to be synced (typically by SbcHeader).
    */
   private async getCurrentAccount (): Promise<any> {
     let account: any
-    for (let i = 0; i < 100; i++) {
+    for (let i = 0; i < 50; i++) {
```

Comment:
The other UIs retry for 5 seconds, which seems to be plenty.

I think, per some of Jonathan's previous work, that we could watch the auth store for this information instead... to be looked at in the future.